dvc/info.py

```diff
@@ -31,34 +31,32 @@ def get_dvc_info():
     ]
 
     try:
-        repo = Repo()
-
-        # cache_dir might not exist yet (e.g. after `dvc init`), and we
-        # can't auto-create it, as it might cause issues if the user
-        # later decides to enable shared cache mode with
-        # `dvc config cache.shared group`.
-        if os.path.exists(repo.odb.local.cache_dir):
-            info.append(
-                "Cache types: {}".format(_get_linktype_support_info(repo))
-            )
-            fs_type = get_fs_type(repo.odb.local.cache_dir)
-            info.append(f"Cache directory: {fs_type}")
-        else:
-            info.append("Cache types: " + error_link("no-dvc-cache"))
-
-        info.append(f"Caches: {_get_caches(repo.odb)}")
-
-        info.append(f"Remotes: {_get_remotes(repo.config)}")
-
+        with Repo() as repo:
```

Comment:
Same here as above, when we are loading info, we should be closing the caches. A lot of warnings/failures were from this and the init.